UIKitTeamPlate/UIKitTeamPlate/CoffeeMenuViewController.swift

```diff
@@ -1,14 +1,309 @@
 // CoffeeMenuViewController.swift
 // Copyright © RoadMap. All rights reserved.
 
-// Наташа
 import UIKit
 
-/// Контролер кофейнного меню
-class CoffeeMenuViewController: UIViewController {
-//    override func viewDidLoad() {
-//        super.viewDidLoad()
-//
-//        // Do any additional setup after loading the view.
-//    }
+/// Протокол для передачи экземпляра модели Coffee между контроллерами после смены опций
+protocol TransferCoffeeOptionsDelegate: AnyObject {
+    /// Метод передачи экземпляра модели Coffee между контроллерами после смены опций
+    func transferCoffeeOptions(item: Coffee)
+}
+
+/// Контролер кофейного меню
+final class CoffeeMenuViewController: UIViewController, UIGestureRecognizerDelegate {
+    // MARK: - Constants
+
+    enum Constants {
+        /// Массив заголовков для использования в сегмент контроллере
```

Comment:
```suggestion
        /// Массив заголовков для использования в сегмент контроллере
```
так же все коменты вычищаем